examples/crypto/README.md

```diff
@@ -0,0 +1,7 @@
+NXP SPSDK Crypto Examples
+=========================
+Here you can find various examples demonstrating **SPSDK** crypto module functionality:
+
+- [key_generation.py](key_generation.py) - demonstrate how to generate RSA keys (private, public key). Required before using examples bellow.
+- [certificates_validation.py](certificates_validation.py) - Demonstrate how to validates previously created chains and their usage.
```

Comment:
maybe to be consistent, use lower case instead..

Reply:
fixed.